### PR TITLE
Add Ice Cream Flavor

### PR DESCRIPTION
**Pull request by jischr**
This PR adds my ice cream flavor to the list of ice cream flavors in `FavoriteIceCreamFlavor/FavoriteIceCreamFlavor.md`

Relevant Issue =  #1

### CODE REVIEW

File: FavoriteIceCreamFlavor/FavoriteIceCreamFlavor.md [hunk 1]
```diff
@@ -10,3 +10,4 @@ Your answer should be in the following format:
 Add your answer below the last entry in this list:
 
 - Cherry Garcia, Vui Nguyen, @vuinguyen
+- Sour Cream and Onion Kettle Chips, Jen Winer, @jenniferwiner
```
Comment (vuinguyen) [inline FavoriteIceCreamFlavor/FavoriteIceCreamFlavor.md]: Are you sure that's not a potato chip flavor? 😄

**Review by vuinguyen — CHANGES_REQUESTED** [review]
Almost ready! Please change that to an ice cream flavor 🍨

File: FavoriteIceCreamFlavor/FavoriteIceCreamFlavor.md (hunk 1 above)
Comment (jischr) [reply FavoriteIceCreamFlavor/FavoriteIceCreamFlavor.md]: oops! good call. will update now!

**Review by vuinguyen — APPROVED** [review]
This looks great! 👍 Good job, @jenniferwiner ! 🎆

**Pull request merged**